{
  "dataType": "CVE_RECORD",
  "dataVersion": "5.1",
  "cveMetadata": {
    "total_count": 1,
    "last_updated": "2025-08-10T05:48:13.756965",
    "severity_distribution": {
      "critical": 0,
      "high": 0,
      "medium": 1,
      "low": 0,
      "none": 0
    }
  },
  "cves": [
    {
      "id": "CVE-2025-8795",
      "publishedDate": "2025-08-10T05:32:06.512Z",
      "lastModifiedDate": "2025-08-10T05:32:06.512Z",
      "description": "A vulnerability, which was classified as critical, was found in LitmusChaos Litmus up to 3.19.0. This affects an unknown part of the file /auth/login. The manipulation of the argument projectID leads to improper access controls. It is possible to initiate the attack remotely. The exploit has been disclosed to the public and may be used. The vendor was contacted early about this disclosure but did not respond in any way.",
      "severity": 6.3,
      "references": [
        {
          "url": "https://vuldb.com/?id.319323",
          "type": "reference"
        },
        {
          "url": "https://vuldb.com/?ctiid.319323",
          "type": "reference"
        },
        {
          "url": "https://vuldb.com/?submit.625984",
          "type": "reference"
        },
        {
          "url": "https://github.com/MaiqueSilva/VulnDB/blob/main/readme05.md",
          "type": "poc"
        }
      ],
      "affected": [
        {
          "vendor": "LitmusChaos",
          "product": "Litmus",
          "versions": [
            {
              "version": "3.0",
              "status": "affected"
            },
            {
              "version": "3.1",
              "status": "affected"
            },
            {
              "version": "3.2",
              "status": "affected"
            },
            {
              "version": "3.3",
              "status": "affected"
            },
            {
              "version": "3.4",
              "status": "affected"
            },
            {
              "version": "3.5",
              "status": "affected"
            },
            {
              "version": "3.6",
              "status": "affected"
            },
            {
              "version": "3.7",
              "status": "affected"
            },
            {
              "version": "3.8",
              "status": "affected"
            },
            {
              "version": "3.9",
              "status": "affected"
            },
            {
              "version": "3.10",
              "status": "affected"
            },
            {
              "version": "3.11",
              "status": "affected"
            },
            {
              "version": "3.12",
              "status": "affected"
            },
            {
              "version": "3.13",
              "status": "affected"
            },
            {
              "version": "3.14",
              "status": "affected"
            },
            {
              "version": "3.15",
              "status": "affected"
            },
            {
              "version": "3.16",
              "status": "affected"
            },
            {
              "version": "3.17",
              "status": "affected"
            },
            {
              "version": "3.18",
              "status": "affected"
            },
            {
              "version": "3.19.0",
              "status": "affected"
            }
          ]
        }
      ],
      "problemType": [
        "Improper Access Controls",
        "Incorrect Privilege Assignment"
      ],
      "fix_suggestion": "无法生成修复建议"
    }
  ]
}